ArduPlane/altitude.cpp

```diff
@@ -358,26 +358,43 @@ int32_t Plane::calc_altitude_error_cm(void)
 }
 
 /*
-  check for FBWB_min_altitude_cm violation
- */
-void Plane::check_fbwb_minimum_altitude(void)
+  check for min and max altitude: FBWB_min_altitude_cm and fbwb_max_altitude_m violation
+*/
+void Plane::check_fbwb_altitude(void)
 {
-    if (g.FBWB_min_altitude_cm == 0) {
+    bool should_check_max = (g.fbwb_max_altitude_m * 100 > g.FBWB_min_altitude_cm && g.fbwb_max_altitude_m > 0); //larger than alt min AND fbwb_max_altitude_m > 0
```

Comment:
I wonder if we should have some minimum alt difference between min and max. Maybe 5 or 10 meters.

Reply:
@IamPete1 Yeah, 10 would be great to have as a difference.